src/workers/notifications/utils.ts

```diff
@@ -25,6 +31,33 @@ export const uniquePostOwners = (
     (userId) => userId && !ignoreIds.includes(userId),
   );
 
+export const getSubscribedMembers = (
+  con: DataSource,
+  type: NotificationType,
+  referenceId: string,
+  where: ObjectLiteral,
+) =>
+  con
+    .getRepository(SourceMember)
+    .createQueryBuilder('sm')
+    .select('"userId"')
+    .where(where)
+    .andWhere(
+      `
```

Comment:
Please use typeorm query builder for this purpose. It's not recommended to use raw queries. We should use it as last resort

Reply:
Finally found how to work it out 🚀